core-services/prow/02_config/openshift/cluster-monitoring-operator/_prowconfig.yaml

```diff
@@ -68,6 +68,25 @@ tide:
     - needs-rebase
     repos:
     - openshift/cluster-monitoring-operator
+  - includedBranches:
+    - main
+    - master
+    labels:
+    - approved
+    - docs-approved
```

Comment:
I don't think we should require `docs-appoved` in our case since we don't have user-facing documentation in CMO's repo

Reply:
Unfortunately we cannot support exceptions at this time.

Reply:
:+1: thank you for the clarification

Reply:
the docs ack is not about putting doc into the repo as part of the PR, it is for the docs team to ack that they have created, or are able to create, the product documentation(docs.openshift.com) related to this feature.